src/main/java/rsb/methods/Game.java

```diff
@@ -64,57 +65,52 @@ public String getLastMessage() {
 	 * @param tab The tab to open.
 	 * @return <code>true</code> if tab successfully selected; otherwise
 	 *         <code>false</code>.
-	 * @see #openTab(GameGUI.Tab tab, boolean functionKey)
+	 * @see #openTab(InterfaceTab tab, boolean functionKey)
 	 */
-	public boolean openTab(GameGUI.Tab tab) {
+	public boolean openTab(InterfaceTab tab) {
 		return openTab(tab, false);
 	}
 
 	/**
 	 * Opens the specified tab at the specified index.
 	 *
-	 * @param tab 				The tab to open
-	 * @param functionKey       If wanting to use function keys to switch.
+	 * @param interfaceTab The tab to open
+	 * @param useHotkey If wanting to use hotkeys to switch.
```

Comment:
Include a note in the useHotkey param here about it using the default OSRS hotkey for the tab, and if a player customized the hotkey in their settings it will not work